azkaban-common/src/main/java/azkaban/executor/ExecutionControllerUtils.java

```diff
@@ -230,7 +230,12 @@ public static void failEverything(final ExecutableFlow exFlow) {
           continue;
           // case UNKNOWN:
         case READY:
-          node.setStatus(Status.KILLING);
+          // if flow status is EXECUTION_STOPPED due to e.g. pod failure, set sub node to KILLED
+          if (exFlow.getStatus()==Status.EXECUTION_STOPPED) {
```

Comment:
Wondering if there were any issues around around jobs showing up in 'running' (something not at 'Ready' state) and will those also be handled by this change ?

Reply:
It's already handled by the default option, setting these nodes (e.g. RUNNING) to FAILED.